### ISSUE TITLE
🐞 - Attached PDF file rendered in iframe, gets reloaded multiple time on page / editor interaction

### ISSUE THREAD

**Issue by hatsantos** [taiga-family/editor#1106]
### Description

When using the latest tui-editor 1.38.0 and the attach tool to add a file, for example a PDF, and render it inline using an iframe, the iframe gets refreshed/reloaded in every mouse move or page/container resize. A lot of http requests are made and we can see a visual flick caused from the reloads.

Also, it's not possible to interact with the iframe content, in this case the PDF controls like zoom, page selection, etc.

I don't know if this is exactly the best or the correct way to render an inline PDF using tui-editor.

```

<tui-editor #internalNotesEditor
	[formControl]="FormControlInternalNotes"
	[tools]="TextEditorSimpleTools"
	(fileAttached)="OnFilesAttached($event, internalNotesEditor)">
</tui-editor>

FormControlInternalNotes = new FormControl("");

OnFilesAttached(files: TuiEditorAttachedFile<IFileUpload>[], editor: TuiEditorComponent)
{
          files.forEach(file =>
          {
	          editor?.editor?.getOriginTiptapEditor().commands.insertContent
	          (
		          `<p><strong>Attachment: </strong><a href="${file.link}" download="${file.name}">${file.name}</a></p>`
	          );
          
	          if (!environment.production)
	          {
		          editor?.editor?.setIframe(
		          {
			          allowfullscreen: false,
			          frameborder: null,
			          
			          //URL hardcoded for testing proposes
			          src: `https://mozilla.github.io/pdf.js/web/viewer.html?url=https://pdfobject.com/pdf/sample.pdf`,
			          //src: `https://mozilla.github.io/pdf.js/web/viewer.html?url${file.link}`,
			          
			          width: "100%",
			          height: 840,
		          });
	          }
          });
}
```

https://github.com/taiga-family/editor/assets/36542752/1fa74b5a-a1c6-45bd-9c6a-be70a0f84c28

### Angular version

18.0.5

### Taiga UI version

3.84

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [ ] macOS
- [X] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

**Comment by splincode** [comment]
Please, check it on next version, is reproduced?

https://taiga-family.github.io/editor/next/

**Comment by hatsantos** [comment]
I'm sorry, how can I install the 2.0.21-alpha.6 version in my project? The 2.0.21-alpha.6 version doesn't seem to be published on npm.

**Comment by splincode** [comment]
<img width="568" alt="image" src="https://github.com/taiga-family/editor/assets/12021443/3dd52d84-0bbb-47ed-9ea7-62e2fd3b3e0e">

**Comment by splincode** [comment]
I checked, the bug is reproducible

**Comment by splincode** [comment]
Fixed by `@tinkoff/tui-editor` v1.39
